tests/workload/apis.go

```diff
@@ -30,3 +31,15 @@ func GetNetwork(ctx context.Context, require *require.Assertions, uris []string,
 		require.Equal(expectedChainID, chainID)
 	}
 }
+
+func GetABI(ctx context.Context, require *require.Assertions, uris []string, expectedABI abi.ABI) {
+	for _, uri := range uris {
+		client := jsonrpc.NewJSONRPCClient(uri)
+		actualABI, err := client.GetABI(ctx)
+		require.NoError(err)
+
+		require.GreaterOrEqual(len(actualABI.Actions), 1)
+		require.NotEmpty(actualABI.Actions[0].Action)
```

Comment:
I’m strongly against checking against a computed value just to avoid tests that pass by checking `nil == nil`. This rule comes from many hours spent dealing with false-positive tests.